flight/Modules/ManualControl/transmitter_control.c

```diff
@@ -1101,6 +1099,9 @@ static void update_stabilization_desired(ManualControlCommandData * manual_contr
                                                STABILIZATIONDESIRED_STABILIZATIONMODE_ATTITUDELQG,
                                                STABILIZATIONDESIRED_STABILIZATIONMODE_LQG };
 
+	const uint8_t FLIPOVER_SETTINGS[3] = {  STABILIZATIONDESIRED_STABILIZATIONMODE_MANUAL,
+                                       STABILIZATIONDESIRED_STABILIZATIONMODE_MANUAL,
```

Comment:
The first column needs to be a tab, else the alignment is baroque :stuck_out_tongue:

Reply:
It matches what's in the file already.

Reply:
:(